rules/style.js

```diff
@@ -121,7 +121,11 @@ module.exports = {
     "operator-linebreak": [2, "after"],
     // Enforce padding within blocks
     // http://eslint.org/docs/rules/padded-blocks
-    "padded-blocks": [2, "never"],
+    "padded-blocks": [2, {
+      "blocks": "never",
+      "classes": "always",
```

Comment:
Also, looking at the doc, it looks like eslint now lets us use human readable strings like `error` and `warning` instead of `0` and `1`.